.storybook/preview.ts

```diff
@@ -0,0 +1,14 @@
+import type { Preview } from "@storybook/react";
+
```

Comment:
import our css file here as well. Will come in handy later on
```suggestion
import '../src/index.css';

```